flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/stream/table/DistinctAggregateTest.scala

```diff
@@ -0,0 +1,119 @@
+/*
```

Comment:
We can probably add these to `org.apache.flink.table.api.stream.table.AggregateTest.scala`